### ISSUE TITLE
Classic Start button font doesn't change

### ISSUE THREAD

**Issue by ToruTheRedFox** [Open-Shell/Open-Shell-Menu#608]
**Describe the bug**
When the "Active Title Bar" font is changed in Windows 7 and earlier, the built-in classic Start button's font changes with it. Open-Shell's classic Start button doesn't do that, and instead remains as Tahoma.

**To Reproduce**
1. Go to "Change window colors and metrics" in Windows 7 or earlier, or copy the cpl from said version to launch it under Windows 8 and later (rename it first so it doesn't invoke its replacement)
2. Change the font for Active Title Bar
3. Click "Apply"

**Version:**
 - Open-Shell: 4.4.160
 - OS: Windows 7 6.1.7601 (Service Pack 1)

**Comment by bonzibudd** [comment]
Yes, I agree with this. I also think that the font should be able to deviate from the system title bar, with a new option to customize it under "Start button" settings.
